### DISCUSSION TITLE
How can I update the avatar in auth users?

### DISCUSSION THREAD

**Discussion by benjyhuub** [supabase/supabase#30203] [Questions]
Hello, I tried to update the picture and avatar_url in raw_user_meta_data, but the image still does not appear in the auth user's avatar. Is there another way or fix to make the avatar_url appear in the avatar?
![Untitled](https://github.com/user-attachments/assets/7c6bf045-1459-4035-b8d9-a8919388aa67)

**Comment by GaryAustin1** [comment]
Have you seen something that says that should ever be an image versus just an icon?

**Reply by benjyhuub** [reply]
> Have you seen something that says that should ever be an image versus just an icon?

Yes, I just called it an icon because it is the size of an icon. But the URL in avatar_url is an image URL
